app/components/header/HeaderKeyStatusSpinner.js

```diff
@@ -1,21 +1,18 @@
 'use strict'
 
-import React, { Component } from 'react'
+import React from 'react'
 
-import colors from '../../assets/styles/variables/colors'
+// import colors from '../../assets/styles/variables/colors'
 
-class HeaderKeyStatusSpinner extends Component {
-
-  render() {
-    return (
-      <div>
-        <div className="spinner">
-          <div className="double-bounce1"></div>
-          <div className="double-bounce2"></div>
-        </div>
+const HeaderKeyStatusSpinner = () => {
```

Comment:
the existing eslint config prefers stateless functional components and it does seem to make sense in this case, of course you could just change the rule if you don't like it this way